api/swagger.yaml

```diff
@@ -210,6 +210,32 @@ definitions:
       PathInContainer: "/dev/deviceName"
       CgroupPermissions: "mrw"
 
+  DeviceRequest:
+    type: "object"
+    description: "A request for devices to be sent to device drivers"
+    properties:
+      Driver:
+        type: "string"
+      Count:
+        type: "string"
+      DeviceIDs:
+        type: "array"
+        items:
+          type: "string"
+      Capabilities:
+        type: "array"
+        items:
+          type: "array"
+          items:
+            type: "string"
+      Options:
+        type: "object"
+        items:
+          type: "string"
+    example:
```

Comment:
Better to put the example values inline (for each property); that way you're sure the format matches the actual structure; here's a quick attempt;


```yaml
  DeviceRequest:
    type: "object"
    description: "A request for devices to be sent to device drivers"
    properties:
      Driver:
        type: "string"
        example: "nvidia"
      Count:
        type: "string"
        example: "all"
      DeviceIDs:
        type: "array"
        items:
          type: "string"
        example:
          - "1"
          - "3"
          - "GPU-fef8089b-4820-abfc-e83e-94318197576e"
      Capabilities:
        description: |
          A list of capabilities; an OR list of AND lists of capabilities.
        type: "array"
        items:
          type: "array"
          items:
            type: "string"
          example:
            - ["one", "green"]
            - ["two", "purple"]
      Options:
        description: |
          Driver-specific options, specified as a key/value pairs. These options
          are passed directly to the driver.
        type: "object"
        x-nullable: true
        additionalProperties:
          type: "string"
        example:
          "nvidia.kmods": "true"
          "limit_mem": "2GB"
```


Note that it currently doesn't show in the docs, because `containers/create` and `containers/{id}/json` override the example values with a full example (I need to open a PR to fix that), but that would show like this:

<img width="457" alt="Screenshot 2019-03-12 at 23 18 36" src="https://user-images.githubusercontent.com/1804568/54240026-378f0a00-451d-11e9-86cf-13d5c533451d.png">